docs/ui/components/Table.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import { theme, borderRadius } from '@expo/styleguide';
+import React from 'react';
+
+type TableProps = {
+  headers: string[];
+  children: React.ReactNode;
+};
+
+export const Table: React.FC<TableProps> = ({ children, headers }) => (
```

Comment:
I think the `React.FC` is deprecated and might be better to move off from these types?

- https://stackoverflow.com/questions/53885993/react-16-7-react-sfc-is-now-deprecated (SO overflow about this)
- https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/ (detailed explanation)

Reply:
Thank you fo the hint about that, I did know that! 🙂

There is an origin PR which deprecated FC or FunctionalComponent types:
* https://github.com/facebook/create-react-app/pull/8177

Also it looks like in this thread and in the article you have mentioned there is a suggested refactor which looks quite nice, so I will apply that.